src/ALCProxy.Communication/Class1.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
```

Comment:
Please make sure to add the license header for all the new cs files (this applies for other files too). Feel free to copy from an existing file:
https://github.com/dotnet/corefxlab/blob/4cefa703d08c2ddc8f950d9ca211cfde06c8f484/src/System.Collections.Sequences/System/Collections/Sequences/ArrayList.cs#L1-L3

Also, name the file to match the class name (so rename `Class1.cs`)